frontstyle.css

```diff
@@ -0,0 +1,11 @@
+* {
+width: 100%;
+height: 100%;
```

Comment:
height: 100% should only be in the html and body selectors.  This has (hopefully) unintended consequences for how your pages is rendered.